src/main/java/com/github/manolo8/darkbot/core/manager/StatsManager.java

```diff
@@ -299,4 +322,38 @@ public String toString() {
                     "\nAverage=" + ONE_PLACE_FORMAT.format(getAverage());
         }
     }
+
+    private enum BootyKeyType {
+        GREEN(0x54, Stats.BootyKey.GREEN),
+        BLUE(0x58, Stats.BootyKey.BLUE),
+        RED(0x5c, Stats.BootyKey.RED),
+        SILVER(0x60, Stats.BootyKey.SILVER),
+        APOCALYPSE(0x64, Stats.BootyKey.APOCALYPSE),
+        PROMETHEUS(0x68, Stats.BootyKey.PROMETHEUS),
+        OBSIDIAN_MICROCHIP(0x6c, Stats.BootyKey.OBSIDIAN_MICROCHIP),
+        BLACK_LIGHT_CODE(0x70, Stats.BootyKey.BLACK_LIGHT_CODE),
+        BLACK_LIGHT_DECODER(0x74, Stats.BootyKey.BLACK_LIGHT_DECODER),
+        PROSPEROUS_FRAGMENT(0x78, Stats.BootyKey.PROSPEROUS_FRAGMENT),
+        ASTRAL(0x7c, Stats.BootyKey.ASTRAL),
+        ASTRAL_SUPREME(0x80, Stats.BootyKey.ASTRAL_SUPREME),
+        EMPYRIAN(0x84, Stats.BootyKey.EMPYRIAN),
+        LUCENT(0x88, Stats.BootyKey.LUCENT),
+        PERSEUS(0x8c, Stats.BootyKey.PERSEUS);
+
+        private int offset;
+        private final StatsAPI.Key key;
+
+        private BootyKeyType(int offset, StatsAPI.Key key) {
+            this.offset = offset;
+            this.key = key;
```

Comment:
```suggestion
            this.key = StatKey.of(key)
```
If you store a stat key directly, and make the track method use it, you'll avoid the N allocations each tick